packages/office-ui-fabric-react/src/components/Stack/examples/Stack.Vertical.Basic.Example.tsx

```diff
@@ -77,7 +80,7 @@ export class VerticalStackBasicExample extends React.Component<{}, {}> {
         </Stack>
 
         <span>Clickable vertical stack</span>
-        <Stack onClick={this._onClick} padding={10} className={styles.root}>
+        <Stack onClick={this._onClick} className={styles.root} tokens={tokens.tenPaddingStack}>
```

Comment:
Bit out of scope for this PR perhaps but it'd be nice to just use styles instead of manually generating classNames